packages/shared/src/api/postsApi.ts

```
@@ -133,6 +133,42 @@ export const sendPost = async ({
   );
 };
 
+export const editPost = async ({
+  channelId,
+  postId,
+  authorId,
+  sentAt,
+  content,
+}: {
+  channelId: string;
+  postId: string;
+  authorId: string;
+  sentAt: number;
+  content: Story;
+}) => {
+  if (isDmChannelId(channelId) || isGroupDmChannelId(channelId)) {
+    return;
```

Maybe throw an error here? Since this shouldn't be getting called.